FV3/conf/configure.fv3.gnu_docker

```diff
@@ -29,7 +29,8 @@ BACIO_LIB4 = $(NCEPLIBS_DIR)/libbacio_4.a
 SP_LIBd = $(NCEPLIBS_DIR)/libsp_v2.0.2_d.a
 W3EMC_LIBd = $(NCEPLIBS_DIR)/libw3emc_d.a
 W3NCO_LIBd = $(NCEPLIBS_DIR)/libw3nco_d.a
-NCEPLIBS = $(NEMSIO_LIB) $(BACIO_LIB4) $(SP_LIBd) $(W3EMC_LIBd) $(W3NCO_LIBd)
+# NCEPLIBS = $(NEMSIO_LIB) $(BACIO_LIB4) $(SP_LIBd) $(W3EMC_LIBd) $(W3NCO_LIBd)
+NCEPLIBS = -L$(NCEPLIBS_DIR) -lnemsio_d -lbacio_4 -lsp_v2.0.2_d -lw3emc_d -lw3nco_d
```

Comment:
You no longer need the '.a' at the end of the files that you've now explicitly included in NCEPLIBS because you've included the NCEPLIBS_DIR? Looks like you can remove or comment out those variable definitions BACIO_LIB4, SP_LIBd, W3EMC_LIBd, W3NCO_LIBd?

Reply:
I deleted the commented out code. The linker does not need the extension (or any other part of the name) when using the `-L` `-l`flags.